packages/dendron-design-system/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "version": "0.1.0",
+  "license": "MIT",
```

Comment:
can you change this to `GPLv3`?

Reply:
yes sure. 
I copied that license file from one of the other packages but missed this :)